services/backend/src/recommendation_system/recommendation_flow/candidate_generators/alpha/TwoTowerANNGenerator.py

```diff
@@ -1,17 +1,23 @@
-
-from src.recommendation_system.recommendation_flow.candidate_generators.AbstractGenerator import AbstractGenerator
+from src.recommendation_system.recommendation_flow.candidate_generators.AbstractGenerator import (
+    AbstractGenerator,
+)
 from src.data_structures.approximate_nearest_neighbor.two_tower_ann import (
-	get_ANN_recommendations_from_content,
-	get_ANN_recommendations_from_user
+    get_ANN_recommendations_from_content,
+    get_ANN_recommendations_from_user,
 )
 
 
 class TwoTowerANNGenerator(AbstractGenerator):
     def _get_content_ids(self, user_id, limit, offset, _, starting_point):
         if starting_point is not None and starting_point.get("content_id", False):
-            content_ids, scores = get_ANN_recommendations_from_content(starting_point["content_id"], "alpha", limit + offset)
+            content_ids, scores = get_ANN_recommendations_from_content(
+                starting_point["content_id"], "alpha", limit + offset
+            )
         else:
-            content_ids, scores = get_ANN_recommendations_from_user(user_id, "alpha", limit + offset)
+            content_ids, scores = get_ANN_recommendations_from_user(
+                user_id, "alpha", limit + offset
+            )
+        print("Number of candidates for TwoTowerANNGenerator: ", len(content_ids))
```

Comment:
please remove the print statement

Reply:
ok but quick question: why is that an issue? just out of curiosity

Reply:
So that there aren't a ton of print statements in the logs and I can debug if a team has an issue 7x print statements make it hard to read when someone has a problem